### PR TITLE
Start work on a test plan

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis

### CODE REVIEW

File: TESTPLAN.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Manual testing of the GCloud Plugin for Eclipse.
+
+# Install the plugin
+
+1. Launch Eclipse 4.5.
+
+2. If necessary, remove the old version of the plugin.
```
Comment (briandealwis) [inline TESTPLAN.md]: Figure out what should happen if we install the new version over top of the old plugin.

File: TESTPLAN.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  2. Click "Google Cloud Platform for Eclipse 4.5". (If this is not present,
+    move on to step 3 below.)
+  3. Click the Uninstall... button.
+  4. Click the Finish button.
+  5. Click the "Yes" button in the dialog that prompts you to restart Eclipse.
+2. ????
+3. Restart Eclipse
+4. Check installation details. 
+
+# Create a project
```
Comment (briandealwis) [inline TESTPLAN.md]: These are related to single project situations.  We'll need something similar for EAR-style multi-service projects.

Comment (elharo) [inline TESTPLAN.md]: agreed

File: TESTPLAN.md (hunk 1 above)
Comment (elharo) [inline TESTPLAN.md]: Ideally nothing special. Mostly this is to make sure we actually are using the new version we're testing and that the install works.

**Comment by elharo** [comment]
PTAL

**Comment by briandealwis** [comment]
LGTM.

**Pull request merged**